### ISSUE TITLE
display works but not touch input

### ISSUE THREAD

**Issue by saehay** [goodtft/LCD-show#163]
Hello,

I have connected and installed the touchscreen and drivers (LCD35) per your instruction manual. The display is working and shows my GUI desktop but I can no longer use my wired keyboard AND the touch input for the screen doesn't work.

**Comment by idamage** [comment]
i have the same issue.

**Comment by TheRealVira** [comment]
Same issue here too on the LCD5-show version. I am trying to get Octoprint with touchscreen to run.

**Comment by TheRealVira** [comment]
@goodtft

**Comment by dhaneshk** [comment]
i have the same problem. display works fine, keyboard and mouse works fine. The touch doesn't. i tried with the provided stylus and fingers.

**Comment by goodtft** [comment]
Please check that the touch IC XPT2046 pins is shorted or not.
You can try to reboot the raspberry pi.

**Comment by just4give** [comment]
Hi,
I am having same problem. Tried rebooting few times. But I don't know how to make sure XPT2046 is shorted or not. Can anyone guide me please?

**Comment by sonicx10** [comment]
Hi,
i use this manual. https://www.root42.de/blog/?p=6
I have successfully resolved.

**Comment by just4give** [comment]
Thanks @sonicx10 ... I will give it a try ...

**Comment by DamageHero** [comment]
> Thanks @sonicx10 ... I will give it a try ...

can you tell me if it worked? i sent returned mine already.

**Comment by paolocolourblind** [comment]
Nope. did not work on mine. Is there a test to determine if this is a kernel issue or hardware?

**Comment by bmelim** [comment]
I have got mine to work and created some instructions. Also used someone else script and edited to make installing easier. 

https://github.com/bmelim/raspi-lcd5-touchscreen

**Comment by srianbury** [comment]
Was having the same issue on RPI 3B+, but it's working now.
1. I first ran the MHS35B-show which displayed on the screen but the touch screen didnt work
2. I tried the script @bmelim provided and neither the display nor the touch screen worked
3. Tried MHS35-show and display and touchscreen are now working

**Comment by fmaglia** [comment]
I have a similar problem.
The keyboard is working, but not the touchscreen.
I used the preparred iso, available on the tutorial website.
I installed it on a raspberry pi 3b+.
Can anyone try to help me to solve this problem?
